src/__tests__/components/explanation/CfFeedbackResulttTable.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {DataTable, TableBody, TableColumn, TableHeader, TablePagination, TableRow} from 'react-md/lib/DataTables/index';
+import CfFeedbackResulttTable from '../../../components/explanation/CfFeedbackResulttTable';
```

Comment:
Just a side note, this file has three 't', I guess 'CfFeedbackResulttTable' -> 'CfFeedbackResultTable'